app/src/pages/domain/index.js

```diff
@@ -1,47 +1,85 @@
-const {ipcRenderer, remote} = window.require('electron')
+import React, {Component, PropTypes} from 'react'
+import DocumentTitle from 'react-document-title'
+
 import css from 'raw!./index.css'
 
-function render() {
-  document.body.innerHTML = `
-    <style type="text/css">
-      ${css}
-    </style>
-    <div class="container">
-      <header>
-        <img class="logo" src="../images/grape-logo.png" alt="Grape">
-      </header>
-      <form class="form">
-        <h1 class="title">Where do you want to connect?</h1>
-        <input class="cbox" id="grape-cbox" type="radio" checked name="chooser"
-        ><label class="tab tab_left" for="grape-cbox">Grape</label
-        ><input class="cbox" id="on-premise-cbox" type="radio" name="chooser"
-        ><label class="tab tab_right" for="on-premise-cbox">On Premise</label>
-
-        <div class="host">
-          <label class="host__label" for="host">Server URL</label>
-          <input class="input" id="host" placeholder="example.com"/>
-        </div>
-        <div class="submit">
-          <button class="submit-btn" type="submit">Continue</button>
-        </div>
-      </form>
-    </div>
-  `
+const {ipcRenderer, remote} = window.require('electron')
+const {domain} = remote.getGlobal('host')
+const {domain: grapeDomain} = remote.getGlobal('grapeHost')
+
+export default class Domain extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      tab: 'grape',
+      value: domain === grapeDomain ? '' : domain
+    }
+  }
 
-  const form = document.querySelector('.form')
-  const input = form.querySelector('#host')
-  const grapeCbox = form.querySelector('#grape-cbox')
-  const onPremiseCbox = form.querySelector('#on-premise-cbox')
-  const domain = remote.getGlobal('host').domain
-  const grapeDomain = remote.getGlobal('grapeHost').domain
+  onSelectOnPremise = () => {
+    this.input.focus()
+    this.setState({tab: 'onPremise'})
+  }
 
-  if (domain !== grapeDomain) input.value = domain
+  onSelectGrape = () => {
+    this.setState({
+      tab: 'grape',
+      value: ''
+    })
+  }
```

Comment:
so if you select 'on Premise' there will be `domain` in input value in case if `domain === grapeDomain ? '' : domain` and then when you switch to 'grape' and back to 'on Premise' it will be empty.